.codecov.yml

```diff
@@ -11,3 +11,6 @@ coverage:
         target: auto
         threshold: 0%
     changes: false
+ignore:
+  - "pkg/client"
+  - "**/zz_generated*"
```

Comment:
there are two small generated telemetry files that end in _generated that may not be filtered out here.

- internal/telemetry/data_attributes_generated.go
- internal/telemetry/nicresourcecounts_attributes_generated.go

Maybe we can add in "internal/telemetry/*generated

Reply:
Thanks @j1m-ryan I've switched it to ignore any files with generated in the name